addon/styles/_frost-sidebar.scss

```diff
@@ -1,62 +1,52 @@
-.frost-sidebar .container {
-  width: 100%;
-  height: 100%;
-}
-
 .frost-sidebar {
   position: absolute;
-  left: 0;
+  left: 0px;
+
+  .liquid-container > .liquid-child {
+    width: auto;
+    height: auto;
+  }
+
+  .open-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+  }
+
+  .closed-container {
+    width: 70px;
+    height: 65px;
+  }
+
+  .pod {
+    padding: 25px;
+    // Value of '1' below hides the content behind this pod completely.
+    background-color: rgba($frost-color-white, 1);
+    box-shadow: 1px 3px 15px $frost-color-grey-5;
+    z-index: $frost-z-index-application-bar;
+  }
+
+  .tab {
+    padding-top: 6px;
+    width: 55px;
+    height: 60px;
+    border-radius: 4px;
+    cursor: pointer;
+    box-shadow: 1px 2px 5px $frost-color-grey-5;
+    background-color: $frost-color-white;
+    z-index: $frost-z-index-application-bar;
```

Comment:
Sorry no - z-index should reference a modal/popup instead of the application bar https://github.com/ciena-frost/ember-frost-core/blob/master/addon/styles/_z-index.scss#L3